src/components/DetailedProduct.js

```diff
@@ -55,33 +55,39 @@ class DetailProduct extends React.Component {
     if (!detailProduct) return null;
     else {
       return (
-        <div>
-          <img
-            src={detailProduct.image}
-            className={'product-image'}
-            alt="Product Image"
-          />
-          <button
-            onClick={() => handleBuy(createItem, matchId, updatedQuantity())}
-          >
-            Buy this stuff!
-          </button>
-          <select onChange={e => updatedQuantity(true, e.target.value)}>
-            {populateQuantityOptions(detailProduct.stock)}
-          </select>
-
-          <h1>{detailProduct.name}</h1>
-          <div>{detailProduct.description}</div>
-          <div>
-            INSTOCK | {detailProduct.instock ? 'YES' : 'NO'} | Available:{' '}
-            {detailProduct.stock}
+        <div className='detailedProducts'>
+          <div id='productTile'>
+            <img
+              src={detailProduct.image}
+              className={'product-image'}
+              alt="Product Image"
+            />
+            <button
+              onClick={() => handleBuy(createItem, matchId, updatedQuantity())}
+            >
+              Buy this stuff!
+            </button>
+            <select onChange={e => updatedQuantity(true, e.target.value)}>
+              {populateQuantityOptions(detailProduct.stock)}
+            </select>
+          </div>
+          <div id='productDetails'>
+            <h1>{detailProduct.name}</h1>
+            <div>{detailProduct.description}</div>
+            <br/>
+            <div>
+              Available:{' '}
+              {detailProduct.stock}
+            </div>
+            </div>
+          <div id='reviewProduct'>
+            <footer>
+              <h4>Reviews for the {detailProduct.name}</h4>
+              <Reviews product={detailProduct} productId={matchId} />
+            </footer>
```

Comment:
I think HTML semantics suggest that a footer actually goes at the bottom of the document, but I think this is ok